src/inlineErrorReporter.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import { ErrorReport } from './SplunkErrorInstrumentation';
+import { INLINE_VERSION } from './version';
+
+interface SplunkRumInlineType {
+  popCapturedErrors: () => ErrorReport[];
+  shutdown: () => void;
+  version: typeof INLINE_VERSION;
+}
+
+declare global {
+  interface Window {
+    __SplunkRumInline?: SplunkRumInlineType;
+  }
+}
+
+const store: ErrorReport[] = [];
+function report(source: ErrorReport['source'], arg: ErrorReport['arg']) {
+  store.push({ source, arg });
```

Comment:
I would add some kind of attribute to errors reported by inline reporter so you could tell that it is from inline reporter

Reply:
Done, but something's broken now, because tests added for it are failing

Reply:
passing Event to `report` -> `reportErrorEvent` -> `report` lost the [extraAttrs so added it](https://github.com/signalfx/splunk-otel-js-web/pull/161/commits/11cfafca12eea628d086a008a09b3c5502e7a797#diff-940884dbc6718136929c55cb91e01215c2f1f2b19b3c7d020aacf50780a9d82e), also while testing locally there was quite a lot of racing happening where full agent started before error was registered by inline handler so updated those tests to load full agent after error happened for sure

Reply:
me: for sure
CI: And then, get this, he thought it would work for sure!